input/europe/documentation

```diff
@@ -0,0 +1,9 @@
+eu_very_crude
```

Comment:
By convention, one typically calls this file "README" or README.txt or README.md rather than "documentation".

Also, can you think of how to include this information in the files themselves rather than in a separate document? (hint, make_reactors.py could be a proper module with functions and they could have proper docstrings.) Keeping the documentation in the files allows 1) allows automated documentation with sphinx, 2) enables dynamic querying with ipython, and 3) is easier to keep up to date. 

Anyway, a readme is great, but by convention it should be called readme. And, it can be made unnecessary by including docstrings. Please see the documentation chapter of the book.